src/content/docs/guides/receive-test-payments.mdx

```diff
@@ -38,10 +38,12 @@ If your browser extension is connected to a wallet that uses real money and you
 
 ## Receive a one-time payment
 
+<Steps>
     1. Go to the <LinkOut href="https://webmonetization.org/play/">Web Monetization Playground</LinkOut>.
     2. Enter your wallet account's payment pointer into the field and select **Add monetization link**.
         :::note[Forgot your payment pointer?]
         Open the extension and select the cog icon. Then, select the **Wallet** tab and copy your payment pointer from the **Connected wallet address** field.
         :::
     3. Open the extension, enter a one-time payment amount, then click **Send now**. A new event will appear in the Playground almost immediately.
-    4. Sign in to your <LinkOut href="https://wallet.interledger-test.dev/">Interledger test wallet</LinkOut> and select **Transactions** from the left menu. A completed outgoing and incoming payment will appear.<Card title="Incoming payment amount">To mimic how a real wallet provider could act, the Interledger test network deducts a 0.01 fee from each outgoing payment. If you made a one-time payment of €1.00 <abbr title="euros">EUR</abbr>, the completed incoming payment will be for €0.99.</Card>
\ No newline at end of file
+    4. Sign in to your <LinkOut href="https://wallet.interledger-test.dev/">Interledger test wallet</LinkOut> and select **Transactions** from the left menu. A completed outgoing and incoming payment will appear.<Card title="Incoming payment amount">To mimic how a real wallet provider could act, the Interledger test network deducts a 0.01 fee from each outgoing payment. If you made a one-time payment of €1.00 <abbr title="euros">EUR</abbr>, the completed incoming payment will be for €0.99.</Card>
```

Comment:
```suggestion
    4. Sign in to your <LinkOut href="https://wallet.interledger-test.dev/">Interledger test wallet</LinkOut> and select **Transactions** from the left menu. A completed outgoing and incoming payment will appear.<Card title="Incoming payment amount">To mimic how a real wallet provider could act, the Interledger test network may deduct a 0.01 fee from each outgoing payment. If you made a one-time payment of €1.00 <abbr title="euros">EUR</abbr>, the completed incoming payment will be for €0.99.</Card>
```

Reply:
That was my assumption as well, but maybe this was changed recently with Rafiki updates?
![image](https://github.com/user-attachments/assets/413ff761-9dcb-4028-9303-c4e22158ffd2)